ivy/array/creation.py

```diff
@@ -79,6 +79,50 @@ def full_like(
             an array having the same shape as ``self`` and where every element is equal
             to ``fill_value``.
 
+        Functional Examples
+        -------------------
+
+        With int datatype:
+            >>> x = ivy.array([1,2,3])
+            >>> fill_value = 0
+            >>> ivy.full_like(x , fill_value)
```

Comment:
you may change this to `x.full_like(fill_value)`  as we don't need functional examples here

Reply:
then replace the title with `Instance Method Examples` from below